modules/ROOT/pages/clustering/setup/routing.adoc

```diff
@@ -120,12 +120,8 @@ Server-side routing driver configuration::
 Server-side routing uses the Neo4j Java driver to connect to other cluster members.
 This driver is configured with settings of the format:
 +
-* <<config_dbms.routing.driver.api, `dbms.routing.driver.*`>>
-+
-[NOTE]
-====
-The configuration options described in _Configuration_ in the link:{neo4j-docs-base-uri}[Neo4j Driver manuals] have an equivalent in the server-side routing configuration.
-====
+*
+link:{neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_dbms.routing.driver.connection.connect_timeout[`dbms.routing.driver.*`]  // the first of driver settings in that list
```

Comment:
The line break is causing the a malformed list:

```suggestion
* link:{neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_dbms.routing.driver.connection.connect_timeout[`dbms.routing.driver.*`]  // the first of driver settings in that list
```

Reply:
I'm not sure whether all readers will understand the use of `//` (and I also wasn't sure what list of driver settings this was referring to?). 

Perhaps a structure like this might be easier for users to understand:

- `dbms.routing.driver.<driver setting>`